### PR TITLE
type: [fix], message: Fix isSetNewVersion function logic.

### PR DESCRIPTION
**Pull request by esequielvirtuoso**
## What?

[Here goes your feature/bug-fix proposal](type: [fix], message: Fix isSetNewVersion function logic.)

### Type of change?

- [ ] [breaking change(s)]: Changes that will require other changes in dependant applications.
- [ ] [build]: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
- [ ] [ci]: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
- [ ] [docs]: Documentation only changes
- [ ] [feat]: A new feature
- [x] [fix]: A bug fix
- [ ] [perf]: A code change that improves performance
- [ ] [refactor]: A code change that neither fixes a bug nor adds a feature
- [ ] [style]: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
- [ ] [test]: Adding missing tests or correcting existing tests

Note: Remove the non-checked options

## Why?

The `isSetNewVersion` function behavior was wrong. It was returning "" while it should return the tag version as a string.

## How?

By returning `fmt.Sprintf("%d.%d.%d", latest.Major, latest.Minor, latest.Patch)`.

## How has this been tested?

- [x] Unit tests
- [x] Integration tests
- [ ] Benchmark tests

# Before submitting a PR please make sure to check if:

- [x] The code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] The changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and/or existing unit tests pass locally with my changes
- [x] New and/or existing integration tests pass locally with my changes

### CODE REVIEW

**Comment by ebbauer** [comment]
LGTM!

**Review by ebbauer — APPROVED** [review]
nice!

File: src/git/git_integration_test.go [hunk 1]
```diff
@@ -346,3 +347,20 @@ func TestNewGitGetCurrentVersionFromRepoWithGreatNumbersTag(t *testing.T) {
 	tests.AssertEqualValues(t, "3.0.2", result)
 	f.cleanLocalRepo(t)
 }
+
+func TestNewGitGetCurrentVersionFromRepoWithDisorderedTags(t *testing.T) {
```
Comment (esequielvirtuoso) [inline src/git/git_integration_test.go:351]: @julianogalgaro 

This is the test I've added to cover disordered tag numbers.

Comment (julianogalgaro) [reply src/git/git_integration_test.go:351]: Nice job!

**Review by julianogalgaro — APPROVED** [review]
Nice!

File: src/git/git_integration_test.go (hunk 1 above)
Comment (julianogalgaro) [reply src/git/git_integration_test.go:351]: Now you can mark as done the requirement:

“I have added tests that prove my fix is effective or that my feature works“

**Pull request merged**